core/src/main/resources/jenkins/console/Messages.properties

```
@@ -1,2 +1,3 @@
 consoleUrlProviderDisplayName=Console URL Provider
+consoleUrlProviderDisplayName.Description=Select which view you want to use for build console links. Tools listed higher will be prioritized over those listed lower. The tools you choose here will be used rather than those set by administrators.
```

maybe? tools doesn't sound right

```suggestion
consoleUrlProviderDisplayName.Description=Select which view you want to use for build console links. Views listed higher will be prioritized over those listed lower. The views you choose here will be used rather than those set by administrators.
```